pyproject.toml

```diff
@@ -59,7 +59,7 @@ click = "^8.0.1"
 commitizen = "^2.20.3"
 flake8 = "^4.0.0"
 ipykernel = "^6"
-ipython = "^7.27.0"
+ipython = ">=7.27.0,<9"
```

Comment:
`<9` is intentional here?

Reply:
Yep, ipython 8 can be used only with python 3.8